### PR TITLE
[OTE-752] Add totalVolume and isWhitelistAffiliate to wallets table

### PR DESCRIPTION
**Pull request by jerryfan01234**
### Changelist
[Describe or list the changes made in this PR]

### Test Plan
[Describe how this PR was tested (if applicable)]

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If the PR has breaking postgres changes to the indexer add the `indexer-postgres-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

## Summary by CodeRabbit

- **New Features**
  - Introduced new properties `totalVolume` and `isWhitelistAffiliate` across various wallet-related functionalities to enhance wallet management and reporting capabilities.
  - Updated query configurations to allow filtering based on wallet affiliate status.

- **Bug Fixes**
  - Enhanced test cases to reflect the new wallet attributes, ensuring correct functionality and validation.

- **Documentation**
  - Updated documentation to include new properties in wallet-related interfaces and models, facilitating better understanding of wallet data structures.

- **Chores**
  - Systematic updates made across tests to align with new wallet object structures, improving overall test coverage and accuracy.

### CODE REVIEW

File: indexer/services/comlink/__tests__/controllers/api/v4/transfers-controller.test.ts [hunk 1]
```diff
@@ -824,6 +834,8 @@ describe('transfers-controller#V4', () => {
       await WalletTable.create({
         address: testConstants.defaultWalletAddress,
         totalTradingRewards: '0',
+        totalVolume: '0',
+        isWhitelistAffiliate: false,
```
Comment (Christopher-Li) [inline indexer/services/comlink/__tests__/controllers/api/v4/transfers-controller.test.ts:838]: nit: these test object are getting big, can you either use defaultWallet or create a constant in the test that is referenced. Please apply across all test files

File: indexer/services/comlink/__tests__/controllers/api/v4/transfers-controller.test.ts [hunk 2]
```diff
@@ -27,6 +27,13 @@ import {
 } from '@dydxprotocol-indexer/postgres/build/__tests__/helpers/constants';
 import Big from 'big.js';
 
+const defaultWallet = {
```
Comment (Christopher-Li) [inline indexer/services/comlink/__tests__/controllers/api/v4/transfers-controller.test.ts:30]: nit: why not use testConstants.defaultWallet imported from postgres?

Comment (jerryfan01234) [reply indexer/services/comlink/__tests__/controllers/api/v4/transfers-controller.test.ts:30]: good point

**Comment by jerryfan01234** [comment]
@Mergifyio backport release/indexer/v6.x

**Comment by jerryfan01234** [comment]
> > backport release/indexer/v6.x
> 
> #### 🟠 Waiting for conditions to match

Commented on wrong PR, will close packport PR.

**Review by Christopher-Li — APPROVED** [review]

**Pull request merged**